commit a3dfa72dada14952d3477712c758ad6d884173e5
Author: Kevin Kofler <kevin.kofler@chello.at>
Date:   Sun Jun 10 23:41:42 2012 +0200

    #undef TIME_UTC to fix conflict with glibc
    
    TIME_UTC is now defined as a macro, so it needs to be undefined before being
    used in an enum.
    
    This is a safer fix for the build failure.

diff --git a/kstars/indi/indielement.h b/kstars/indi/indielement.h
index cd05717..93be2ba 100644
--- a/kstars/indi/indielement.h
+++ b/kstars/indi/indielement.h
@@ -44,6 +44,8 @@ typedef enum {PG_NONE = 0, PG_TEXT, PG_NUMERIC, PG_BUTTONS,
               PG_RADIO, PG_MENU, PG_LIGHTS, PG_BLOB} PGui;
 
 /* INDI std properties */
+/* new versions of glibc define TIME_UTC as a macro */
+#undef TIME_UTC
 /* N.B. Need to modify corresponding entry in indidevice.cpp when changed */
 enum stdProperties { CONNECTION, DEVICE_PORT, TIME_UTC, TIME_LST, TIME_UTC_OFFSET, GEOGRAPHIC_COORD,   /* General */
                      EQUATORIAL_COORD, EQUATORIAL_EOD_COORD, EQUATORIAL_EOD_COORD_REQUEST, HORIZONTAL_COORD,  /* Telescope */
